charts/prometheus-kafka-exporter/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-appVersion: "v1.2.0"
+appVersion: "v1.3.1"
 description: A Helm chart to export the metrics from Kafka in Prometheus format using the kafka-exporter from https://github.com/danielqsj/kafka_exporter
 name: prometheus-kafka-exporter
 home: https://github.com/danielqsj/kafka_exporter
-version: 1.1.1
+version: 1.1.2
```

Comment:
adding new features should raise charts minor version.

Reply:
Done